src/navs/index.json

```diff
@@ -182,8 +182,8 @@
                                 "title": "Analytics",
                                 "items": [
                                     {
-                                        "title": "Cohorts",
-                                        "url": "/docs/user-guides/cohorts"
+                                        "title": "Trends",
```

Comment:
Reordered here based on relevancy. Also focused only on features that can actually give you useful insights directly.